tests/functional/helpers.js

```diff
@@ -9,31 +9,28 @@ export const getUri = uri => browser.options.baseUrl + uri
 export const getRandomString = length => sampleSize('ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789', length).join('')
 
 export const startCensusQuestionnaire = (schema, sexualIdentity = false, region = 'GB-ENG', language = 'en') => {
-  devPage.open()
-    .setUserId(getRandomString(10))
-    .setCollectionId(getRandomString(10))
-    .setSchema(schema)
-    .setRegionCode(region)
+  generateToken(schema, getRandomString(10), getRandomString(10), null, null, region, language, sexualIdentity)
+    .then(function(token) {
+      browser.url('http://localhost:5000/session?token=' + token)
```

Comment:
Can we get a return here so I can compose this promise with my work in https://trello.com/c/7H5ReU7v/1095-refactor-functional-test-page-objects?